js/propExtractorConfig.ts

```diff
@@ -1,5 +1,76 @@
 import { PropExtractorConfig } from './util/extractProps';
 
+const REACT_NATIVE_ELEMENTS_CONFIG: PropExtractorConfig = {
```

Comment:
This looks great!  I think I might have a slight preference to keep this in its own file and import it into here?  Maybe create a directory with default configs?

Reply:
+1, think this is smart.